src/CommunityToolkit.Maui.Core/Primitives/Defaults/GravatarImageSourceDefaults.shared.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace CommunityToolkit.Maui.Core;
```

Comment:
With removing `IGravatarImageSource` we can move these defaults over to `GravatarImageSource` as they are only needed for that file.

Reply:
All defaults moved over to `GravatarImageSource.shared.cs` and interface removed.